Vagrantfile

```diff
@@ -80,8 +80,8 @@ die() {
 echo "Installing build artifacts on the host"
 
 mkdir -p /srv/scality || die "Failed to create /srv/scality"
-rm -f "/srv/scality/metalk8s-$SHORT_VERSION" || die "Failed to unlink symlink destination"
```

Comment:
The move from `SHORT_VERSION` to `VERSION` clearly deserve a commit (and commit message) on its own.

Reply:
@gdemonet @slaperche-scality I'm gonna remove this change from this PR, you're right, it deserves a new issue/ticket